### ISSUE TITLE
Issues after upgrading to 0.68.5

### ISSUE THREAD

**Issue by DevonWieczorek** [tinacms/tinacms#2914]
## Description

After upgrading to 0.68.5 we have noticed two main issues:

1. There are some minor style issues in both the toolbar up top, as well as the sidebar when filling out a form.
2. Clicking the Expand button and/or the Hamburger Menu button cause the site to crash with the following error:
```text
index.js:71 Uncaught Error: Unable to find `TinaProvider`; did you forget to add the TinaCMS container to your app root?  See our setup docs: https://tina.io/docs/introduction/tina-init/#adding-tina
    at Module.useEditState (index.js:71:13)
    at Nav (index.js:8155:19)
    at renderWithHooks (react-dom.development.js:14969:22)
    at mountIndeterminateComponent (react-dom.development.js:17731:17)
    at beginWork (react-dom.development.js:18930:20)
    at HTMLUnknownElement.callCallback (react-dom.development.js:3919:18)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:3968:20)
    at invokeGuardedCallback (react-dom.development.js:4028:35)
    at beginWork$1 (react-dom.development.js:23770:11)
    at performUnitOfWork (react-dom.development.js:22606:16)
```

### Steps to reproduce

Unfortunately, we cannot provide a demo project, however, this should be very easy to reproduce.

**Issue One:**

- Simply navigate to any page in which the toolbar is enabled. The _three dots_ button in the top right corner that triggers the tooltip to _Exit Tina_ is taller than the toolbar itself and is overflowing.
- Navigate to any editable page in which a form is enabled.
-- When the sidebar is collapsed, the button to open the sidebar is covering the hamburger menu button in the top left.
-- Click the button to expand the sidebar.
--- The title of that page is flowing underneath the expand button.

**Issue Two:**

- Navigate to any editable page in which a form is enabled.
- Click the button to expand the sidebar.
- Click either the hamburger menu on the left of the sidebar or the expand button toward the right of the sidebar.
- Page will crash with the above-attached error.

### Expected result

1. Buttons should not overflow from their container in the toolbar.
2. Text should not flow beneath buttons in the sidebar.
3. Clicking the expand button in the sidebar should (presumably) expand the form to full screen.
4. Clicking the hamburger menu in the sidebar should bring up additional Tina options.

### Actual result

1. The _three dots_ button in the top right corner of the toolbar is overflowing out the toolbar. 
2. The title of the page in the sidebar is flowing beneath the expand button.
3. Clicking the expand button in the sidebar causes the page to crash with the above-attached error.
4. Clicking the hamburger menu in the sidebar causes the page to crash with the above-attached error.

#### Tinacms Package Versions

```text
├── react-tinacms-editor@0.53.19
└── tinacms@0.68.5
```

#### Tools

| Tool    | Version |
| ------- | ------- |
| browser |    Chrome v101.0.4951.64     |
| os      |     MacOS Monterey v12.3.1    |
| node    |    v17.4.0     |
| npm     |     v8.4.0    |

#### Screenshots

Uploading screenshots here does not seem to be working for me for some reason, however, screenshots can be provided via Discord or email.

**Comment by DevonWieczorek** [comment]
<img width="602" alt="Screen Shot 2022-05-18 at 12 12 04 PM" src="https://user-images.githubusercontent.com/16159377/169096914-85436d57-73ab-492d-be89-d64ef17e56a1.png">

**Comment by perkinsjr** [comment]
This may be related to them not using Tina + Tina Cloud or the CLI but using Tina to power forms. User will attempt to revert to https://www.npmjs.com/package/tinacms/v/0.67.4

**Comment by perkinsjr** [comment]
This has been "resolved" they are going to stick with 0.43.3 which is the last version they can use.
